.travis.yml

```diff
@@ -7,39 +8,41 @@ matrix:
       env: TOXENV=py27
     - python: '3.4'
       env: TOXENV=py34
+      dist: trusty
     - python: '3.5'
       env: TOXENV=py35
     - python: '3.6'
       env: TOXENV=py36
     - python: '3.7'
       env: TOXENV=py37
-      dist: xenial
-      sudo: true
     - python: 'pypy'
       env: TOXENV=pypy
+      dist: trusty
     - python: 'pypy3'
       env: TOXENV=pypy3
-    - env: TOXENV=flake8
-    - env: TOXENV=checkspelling
-
-addons:
-  apt:
-    packages:
-    - libtidy-0.99-0
-    - aspell
-    - aspell-en
+      dist: trusty
+    - env: TOXENV=flake8,checkspelling
```

Comment:
I definitely do ***not*** want these combined. I like the granularity of each item being a single thing.

Reply:
Sure, I've kept it in a separate commit for that reason.

Reply:
I mean `flake8` and `checkspelling` should be separate Travis jobs.